WordPressUI/Bottom Sheet/BottomSheetViewController.swift

```diff
@@ -0,0 +1,192 @@
+import UIKit
+
+public class BottomSheetViewController: UIViewController {
+    enum Constants {
```

Comment:
💭 right now the use cases in WCiOS (Products tab sort options example) have the bottom sheet height fit to its content height, so we currently calculate the content height of the child view controller's table view **plus** the [space above (grip view height + stack view spacing + stack view vertical insets)](https://github.com/woocommerce/woocommerce-ios/blob/42981e7c5d28537d1fafad8cd3601e2a59cc21bf/WooCommerce/Classes/ViewRelated/BottomSheet/BottomSheetViewController+Constants.swift#L3-L9) for the [`DrawerHeight.contentHeight` value in collapsed & expanded states](https://github.com/woocommerce/woocommerce-ios/blob/42981e7c5d28537d1fafad8cd3601e2a59cc21bf/WooCommerce/Classes/ViewRelated/BottomSheet/ListSelector/BottomSheetListSelectorViewController+DrawerPresentable.swift#L4-L10).

does WPiOS have a use case where the bottom sheet fit to the content height (like the native action sheet)? what do you all think of these options:
- adding the [height constant above the content (grip view height + stack view spacing + stack view vertical insets)](https://github.com/woocommerce/woocommerce-ios/blob/42981e7c5d28537d1fafad8cd3601e2a59cc21bf/WooCommerce/Classes/ViewRelated/BottomSheet/BottomSheetViewController+Constants.swift#L3-L9) to WordPressUI
  - this constant also works for iPad popover
- adding a new `DrawerHeight` option that fits to the child view controller's content height (I'm not sure if it's similar to [`PanModal`'s `intrinsicHeight` case](https://github.com/slackhq/PanModal/blob/b2f5bd7d169cba564d5c5b7203864f79be3fe826/PanModal/Presentable/PanModalHeight.swift#L42)?) - this might be tricky with child view controller with a table view or a navigation controller 🤔 so far I can only get the table view's height correctly by [calling `layoutIfNeeded` and then getting the `contentSize`](https://github.com/woocommerce/woocommerce-ios/blob/42981e7c5d28537d1fafad8cd3601e2a59cc21bf/WooCommerce/Classes/ViewRelated/BottomSheet/ListSelector/BottomSheetListSelectorViewController.swift#L18-L24)

feel free to suggest any other ideas as well!

other than this, it's working nicely from my testing on iPhone & iPad simulators 💯

Reply:
Added in 40b0fb1 and a00b063